Tabloid/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Tabloid.Models;
+using Tabloid.Repositories;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoryController : ControllerBase
+    {
+        private readonly ICategoryRepository _categoryRepository;
+        public CategoryController(ICategoryRepository categoryRepository)
+        {
+            _categoryRepository = categoryRepository;
+        }
+
+        [Authorize]
+        [HttpGet("Get/All")]
```

Comment:
not really sure what the rationale for this route name is. Feels like `/api/categories` should be enough. If you do go with this, you'll have to stick with this naming scheme for the entire api. Maybe discuss as a group?